models/repositories/invoicePurchaseToDB.php

```diff
@@ -0,0 +1,58 @@
+<?php
+global $config;
+class invoicePurchaseToDB
+{
+    public static function insertToDB($invoiceNumber, $vatID, $amountNett, $amountGross, $amountTAX, $currencyName, $amountNettCurrency,$addDate)
+    {
+        global $config;
+        if(strlen($invoiceNumber)<3){
+            echo "zle dane";
+        }
+        else{
+            echo "dobre dane";
+            $pdo = new PDO($config['dsn'], $config['login'], $config['password']);
+            $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+            $pdoQuery = "INSERT INTO invoicepurchase (invoiceNumber,addDate, vatID, amountNett, amountGross, amountTAX, currencyName, amountNetCurrency)
+        VALUES(:invoiceNumber, :addDate, :vatID, :amountNett, :amountGross, :amountTAX, :currencyName, :amountNetCurrency)";

```

Comment:
Można użyć notacji (?,?,?,...) i przesyłać w execute($params) który byłby tablicą przesyłaną do tej metody jako parametr jest to myśle lepsze rozwiązanie